docs/source/tutorial.rst

```diff
@@ -0,0 +1,126 @@
+Tutorial
+--------
+
+For this tutorial we are going to use a well known case file from `NIST <https://cfreds.nist.gov/all/NIST/HackingCase>`_.
+In this tutorial we assume Linux is used, on other operating systems most steps are the same but details may differ.
```

Comment:
```suggestion
In this tutorial we use a Linux operating system. Most steps will be same if you perform this tutorial on other operating systems but some details may be different.
```

Reply:
I don't really see the improvement here, except that it's more verbose?